hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/OzoneRpcClientTests.java

```diff
@@ -5293,4 +5297,128 @@ public void testGetObjectTagging(BucketLayout bucketLayout) throws Exception {
     assertEquals(tags.size(), tagsRetrieved.size());
     assertThat(tagsRetrieved).containsAllEntriesOf(tags);
   }
+
+  @Test
+  public void testCreateLifecycleConfiguration() throws Exception {
```

Comment:
Can we parameterize these new tests, to cover both OBS and FSO bucket?

Reply:
Added.